src/api/helpers/string/index.test.ts

```diff
@@ -0,0 +1,21 @@
+import { isEmptyString } from '.';
+
+describe('api/helpers/string', () => {
+  describe('isEmptyString', () => {
+    describe('when a string is empty', () => {
+      it('should return true', () => {
```

Comment:
Can you add one for `null` and `undefined` please.

Reply:
The only way to do this would be to add a `ts-ignore` for both `null` and `undefined` because the required type is a string. I don't think we need to do this.

Reply:
Apologies, I didn't notice `string` type.